### ISSUE TITLE
🐛 [BUG]: Multi-Select doesn't work after zooming with snap-to-grid

### ISSUE THREAD

**Issue by delphi-sucks** [bcakmakoglu/vue-flow#1606]
### Is there an existing issue for this?

- [X] I have searched the existing issues and this is a new bug.

### Current Behavior

If the property `snap-to-grid` is enabled and the user zoomes in or out, it's not possible to select multiple nodes with **Ctrl** pressed anymore.

Edges can still be selected with **Ctrl**.

### Expected Behavior

Multi-Select should still work even after zomming in/out with `snap-to-grid` enabled.

### Steps To Reproduce

1. Create a Basic Vue-Flow with at least two nodes and the property `snap-to-grid` enabled
2. Zoom in/out
3. Try select multiple nodes with **Ctrl** pressed
4. `getSelectedElements` will only return the first selected node

[Stackblitz](https://stackblitz.com/edit/vitejs-vite-ofnha2?file=src%2Fcomponents%2FHelloWorld.vue)

```vue
<script setup>
import { ref } from 'vue';
import { VueFlow, useVueFlow, SelectionMode } from '@vue-flow/core';

const { getSelectedElements } = useVueFlow();

const nodes = ref([
  {
    id: '1',
    type: 'input',
    position: { x: 250, y: 5 },
    data: { label: 'Node 1' },
  },

  {
    id: '2',
    position: { x: 100, y: 100 },
    data: { label: 'Node 2' },
  },
  {
    id: '3',
    position: { x: 400, y: 200 },
    data: { label: 'Node 3' },
  },
]);

const edges = ref([
  {
    id: 'e1->2',
    source: '1',
    target: '2',
  },
  {
    id: 'e2->3',
    source: '2',
    target: '3',
  },
]);
</script>

<template>
  <VueFlow
    :nodes="nodes"
    :edges="edges"
    :snap-grid="[1, 1]"
    :snap-to-grid="true"
    style="min-height: 400px"
  >
  </VueFlow>
  Selected Elements Count: {{ getSelectedElements.length }}
</template>

<style>
@import '@vue-flow/core/dist/style.css';
@import '@vue-flow/core/dist/theme-default.css';
</style>
```

### Anything else?

Package versions:
* @vue-flow/core: v1.41.0

**Comment by bcakmakoglu** [comment]
Thanks for the report, I'll check it out as soon as I have some time :)

**Comment by bcakmakoglu** [comment]
Will be fixed in `1.41.1`

**Comment by bcakmakoglu** [comment]
Fixed with `1.41.1`
